lib/id.js

```diff
@@ -50,7 +50,7 @@ const generateGUID = () => {
 };
 
 // Calculate SID CRC
-//   config - record, { length, characters, secret }
+//   config - record, { secret }
```

Comment:
You haven't addressed the second part of the https://github.com/metarhia/common/pull/194#discussion_r219775382:
>Also, the first word after the dash describes the type in all of the other comments, and 'record' here should probably be replaced with an 'object' since this function is expecting an object.